src/main/java/me/moodcat/mood/Mood.java

```diff
@@ -8,19 +10,20 @@
  *
  * @author JeremybellEU
  */
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
 public enum Mood {
 
     // CHECKSTYLE:OFF
-    ANGRY(0.0, 0.0),
-    CALM(0.0, 0.0),
-    EXCITING(0.0, 0.0),
-    HAPPY(0.0, 0.0),
-    NERVOUS(0.0, 0.0),
-    PLEASING(0.0, 0.0),
-    PEACEFUL(0.0, 0.0),
-    RELAXED(0.0, 0.0),
-    SAD(0.0, 0.0),
-    SLEEPY(0.0, 0.0);
+    ANGRY(new VAVector(-1.0, -1.0), "Angry"),
```

Comment:
I can try to determinate this vectors.
We hadn't written them down yet, but that should be feasible.
We just tried to determinate a vector for a song and that was a bit of "nattevingerwerk"/inaccurate.